src/actions/clusterActions.js

```diff
@@ -269,57 +269,61 @@ function clusterLoadStatus(clusterId, { withLoadingFlags }) {
  * @param {Boolean} isV5Cluster
  */
 export function clusterCreate(cluster, isV5Cluster) {
-  return function(dispatch) {
-    dispatch({ type: types.CLUSTER_CREATE_REQUEST });
-
-    const method = isV5Cluster
-      ? 'addClusterV5WithHttpInfo'
-      : 'addClusterWithHttpInfo';
+  return async function(dispatch) {
+    try {
+      dispatch({ type: types.CLUSTER_CREATE_REQUEST });
 
-    return clustersApi[method](cluster)
-      .then(data => {
-        const location = data.response.headers.location;
-        if (typeof location === 'undefined') {
-          throw new Error('Did not get a location header back.');
-        }
+      const method = isV5Cluster
+        ? 'addClusterV5WithHttpInfo'
```

Comment:
I know you didn't introduce this in the PR, but can you please change these to something like:
```javascript
const method = isV5Cluster
  ? clustersApi.addClusterV5WithHttpInfo
  : clustersApi.addClusterWithHttpInfo
```

Having the function names as strings makes it impossible for our code editors to check if we used the wrong names.
One wrong letter and it will break cluster creation for everyone...

Reply:
> I know you didn't introduce this in the PR, but can you please change these to something like:

Actually I think I did! 😂 
Yeah, and this is more straigthforward, changing it 👍